src/test/storage.zig

```diff
@@ -355,6 +371,21 @@ pub const Storage = struct {
             else => unreachable,
         }
 
+        {
+            // Allow at most `f` faulty replicas to ensure the view change can succeed.
+            // TODO Allow more than `f` faulty replicas when the fault is to the right of the highest known replica.op (and to the left of the last checkpointed op).
+            const majority = (replica_count / 2) + 1;
```

Comment:
I've been thinking we should use `@divFloor` and `@divExact` more, to show the reader that we've thought through how we handle even/odd/exact divisions.